src/pages/CategoriesPages.js

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const CategoriesPages = () => (
-  <div>
-    <button type="submit">Check status</button>
-  </div>
-);
+const CategoriesPages = () => {
+  const dispatch = useDispatch();
+
+  const checkCategoriesStatus = () => {
+    dispatch(checkStatus());
+  };
+  return (
+    <div>
+      <h3>This is the categories page</h3>
+      <button type="button" onClick={checkCategoriesStatus}>Check status</button>
+    </div>
+  );
```

Comment:
**[Optional]**

- [ ] It's great that you have a **checkCategoriesStatus** function which is returning a string `Under Construction` 👏 It would be nice if Check Status button is clicked and it shows that `Under Construction` string on your dom 😊

Reply:
Hi @HAMMAS-SALEEM , well said in and  thanks for the suggestion on those particular lines 5 to 16